flocker/provision/_ssh/_crochet.py

```diff
@@ -0,0 +1,28 @@
+from crochet import setup, run_in_reactor
+from effect import ComposedDispatcher
+
+from .._effect import dispatcher
+from ._conch import dispatcher as conch_dispatcher
+from effect.twisted import (
+    make_twisted_dispatcher,
+    perform as perform_with_twisted
+)
+from twisted.internet import reactor
+
+
+def perform(dispatcher, effect):
+    """
+    Perform an effect in a reactor with crochet.
+    """
+    setup()
```

Comment:
I'm only using crochet here because this branch was getting to big, and I didn't want to change all of `admin.acceptance` to be asynchronous in this branch.